### DISCUSSION TITLE
High CPU usage while using aiortc compared to chrome

### DISCUSSION THREAD

**Discussion by harkiratsing** [aiortc/aiortc#746] [Q&A]
Before filing an issue please verify the following:

This is sort of on the lines of https://github.com/aiortc/aiortc/issues/104

We've created a communication application that connects a browser client to a python client.
When we send raw video (480p) from the `python client` to the browser, we see a huge lag in the video and a spike in the CPU usage.

When we send the same size video from browser to browser, we don't get a lag and the cpu usage is manageable. 

We use vp8 encoding in both the cases.

We're running the code on a fairly weak machine (A Raspberry pi 4), but we were hoping to see similar results when sending video from the browser and from the python client. 

Any pointers around what might be causing this/ways to get around this?

**Comment by marcus2vinicius** [comment]
If anyone has an update on this let me know.
